main/preload.js

```diff
@@ -46,6 +50,12 @@ contextBridge.exposeInMainWorld('electron', {
 
       ipcRenderer.on('station:earnings-counter', listener)
       return () => ipcRenderer.removeListener('station:earnings-counter', callback)
+    },
+    onUpdateAvailable: (/** @type {() => void} */ callback) => {
+      const listener = () => callback()
+
+      ipcRenderer.on('station:update-available', listener)
+      return () => ipcRenderer.removeListener('station:update-available', callback)
```

Comment:
```suggestion
      return () => ipcRenderer.removeListener('station:update-available', listener)
```

We have made this mistake above as well, can you fix them as part of this PR too?

Reply:
[5c37fd3](https://github.com/filecoin-station/filecoin-station/pull/199/commits/5c37fd3fd9ae539d4890239b888c6e83928fcbe4)